CHANGELOG.md

```diff
@@ -57,7 +57,7 @@
 * Rely on `abstract-leveldown` defaults in `Iterator` constructor (@vweevers)
 * Rely on `abstract-leveldown` callback defaults (@vweevers)
 * Remove testling from `package.json` (@vweevers)
-* Remove `level.js` logo (@vweevers)
+* Remove `level-js` logo (@vweevers)
```

Comment:
Actually I purposefully called it `level.js` because that's what the logo said ;) But doesn't matter

Reply:
Just made a query replace. I'll fix.